index.js

```diff
@@ -1 +1,25 @@
-// Add your code here
+function submitData(name, email) {
+  const paragraph = document.createElement('p');
+
+  return fetch('http://localhost:3000/users', {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+      "Accept": "application/json"
+    },
+    body: JSON.stringify({
+      'name': name,
+      'email': email
+    })
+  })
+  .then(resp => resp.json())
+  .then(user => {
+    document.querySelector('body').appendChild(paragraph).textContent = user.id
+  })
+  .catch(() => document.querySelector('body').appendChild(paragraph).textContent = 'FetchError: request to http://localhost:3000/users failed, reason: Unauthorized Access')
```

Comment:
While your solution works, it can be broken up to make it more readable, which very important. 
```suggestion
  .catch(() => {
    const body = document.querySelector('body')
    paragraph.textContent = 'FetchError: request to http://localhost:3000/users failed, reason: Unauthorized Access')
    body.appendChild(paragraph)
```